### ISSUE TITLE
`grunt init` not working correctly on Windows

### ISSUE THREAD

**Issue by fatfisz** [flowxo/flowxo-sdk#34]
The output looks like that:
```
Running "flowxo:init" (flowxo) task
Installing SDK dependencies...
{ [Error: spawn npm ENOENT]
  code: 'ENOENT',
  errno: 'ENOENT',
  syscall: 'spawn npm',
  path: 'npm' }
```

The problem seems to be with this line in `tasks/lib/init.js`:
```
spawn('npm', ['install'], {
```

This clearly doesn't work on Windows and is related to the issue with Node.js nodejs/node-v0.x-archive#2318.

Some people in that thread suggested using [cross-spawn](https://www.npmjs.com/package/cross-spawn), which works across multiple platforms. It seems to be quite popular.

**Comment by fiznool** [comment]
Thanks for reporting this. Windows support hasn't been tested with the SDK. Happy to accept a PR as I don't have a Windows box to test/debug/fix this issue!

As an aside I'll make it more clear that Windows isn't really supported in the README.

**Comment by fatfisz** [comment]
If using cross-spawn is okay with you, I'll gladly submit a PR concerning this issue sometime soon.

**Comment by fiznool** [comment]
yes, that will be fine, cross-spawn is very popular.

**Comment by fiznool** [comment]
Fixed via #36
